python/clicastInterface.py

```diff
@@ -354,17 +354,11 @@ def generateExecutionReport(enviroPath, testIDObject):
     # We build a clicast command script to generate the execution report
     # since we need multiple commands
     with open(commandFileName, "w") as commandFile:
+        # we force report mode to HTML just to be safe
+        commandFile.write("option VCAST_CUSTOM_REPORT_FORMAT HTML\n")
```

Comment:
I'm not saying this is better/worse than what we have currently, but won't this leave the environment in the HTML state if the user has selected TEXT?

Reply:
Yes, as it did before.  I think that the number of people who have environments in text mode would be really small,
but we could open the CFG file to check the current value (if any) to improve this code.

I'm happy to do this, but we would have to do this file open/parse on each execute with report, or report command to be perfect.  Or we could do it once for each "base" directory when we load the environments.

When I initially wrote this code, my opinion was that it was not worth the effort.

Reply:
As soon as we implement the new reporting approach, this will all go away anyway, so it really isn't a big deal to me ...